### ISSUE TITLE
Move 3GPP-specific bibdata under `ext`

### ISSUE THREAD

**Issue by ronaldtse** [relaton/relaton-3gpp#11]
We have data like:
```yaml
doctype: TR
editorialgroup:
- name: RP
  type: prime
- name: R1, R2, R3, R4
  type: other
radiotechnology: 3G
release:
  version2g: '7'
  version3g: '7'
  defunct: true
  wpm_code_2g: GSM_Release_7
  wpm_code_3g: 3G_Rel-7
  freeze_meeting: SP-35
  freeze_stage1_meeting: SP-29
  freeze_stage2_meeting: SP-33
  freeze_stage3_meeting: SP-35
  close_meeting: SP-65
  project_start: '2003-12-01'
  project_end: '2008-03-13'
```

But these are 3GPP specific data. They should be stored under a special key and not pollute the top level namespace.

**Comment by ronaldtse** [comment]
I agree that we should store under `ext` element in Relaton YAML too. Thanks!

**Comment by andrew2net** [comment]
fixed in relaton v1.20.0
